core/core_bind.h

```diff
@@ -567,6 +567,28 @@ class EngineDebugger : public Object {
 	~EngineDebugger();
 };
 
+class LogManager : public Object {
+	GDCLASS(LogManager, Object);
+
+	Callable log_capture;
+
+protected:
+	static void _bind_methods();
+	static LogManager *singleton;
+
+public:
+	LogManager();
+	~LogManager();
+
+	static LogManager *get_singleton() { return singleton; }
+
+	void register_log_capture(const Callable &p_callable);
+
+private:
+	friend class ::UserLogManagerLogger;
+	void process(const Dictionary &&p_message);
```

Comment:
```suggestion
	void process(const Dictionary &p_message);
```
Const move doesn't make sense to me, and `Dictionary` isn't designed with move capabilities in mind, we don't use move like this in the engine, and in any case `Dictionary` is ref-counted so it doesn't need to be moved

Reply:
Fixed!